Tella/Data/Database/Dropbox/DropboxDatabase.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DropboxDatabase.swift
+//  Tella
+//
+//  Created by gus valbuena on 9/9/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+extension TellaDataBase {
+    func createDropboxServerTable() {
+        let columns = [
+            cddl(D.cServerId, D.integer, primaryKey: true, autoIncrement: true),
+            cddl(D.cName, D.text)
+        ]
+        
+        statementBuilder.createTable(tableName: D.tDropboxServer, columns: columns)
+    }
+    
+    func addDropboxServer(dropboxServer: DropboxServer) -> Result<Int, Error>{
+        do {
+            let valuesToAdd = [KeyValue(key: D.cName, value: dropboxServer.name)]
+            
+            let serverId = try statementBuilder.insertInto(tableName: D.tDropboxServer, keyValue: valuesToAdd)
+            return .success(serverId)
+        } catch let error {
+            return .failure(error)
```

Comment:
Could we add a generic error to be displayed for user in case of error has occured in database 
for example :

    func addDropboxServer(dropboxServer: DropboxServer) -> Result<Int, Error> {
        do {
            let valuesToAdd = [KeyValue(key: D.cName, value: dropboxServer.name)]
            let serverId = try statementBuilder.insertInto(tableName: D.tDropboxServer, keyValue: valuesToAdd)
            return .success(serverId)
        } catch let error {
            debugLog(error)
            return  .failure(RuntimeError(LocalizableCommon.commonError.localized))
        }
    }

Reply:
@Rimktarii What do you think ?